src/test/java/uk/gov/companieshouse/authcodenotification/validation/AuthCodeEmailValidatorTest.java

```diff
@@ -26,15 +25,10 @@ void setup() {
         authCodeEmailValidator = new AuthCodeEmailValidator();
     }
 
-    @Test
-    void testSuccessfulValidationWhenCompanyNumberStartsWithTwoUppercaseLetters() {
-        Errors errors = authCodeEmailValidator.validate("OE000001", "A1B2C3", new Errors(), CONTEXT);
-        assertFalse(errors.hasErrors());
-    }
-
-    @Test
-    void testSuccessfulValidationWhenCompanyNumberStartsWithTwoLowercaseLetters() {
-        Errors errors = authCodeEmailValidator.validate("oe001100", "A1B2C3", new Errors(), CONTEXT);
+    @ParameterizedTest
+    @ValueSource(strings = { "00110011", "ABCDEFGH","OE000001", "oe001100"})
+    void testSuccessfulValidationWhenCompanyNumberStartsWithTwoLowercaseLetters(String companyNumber) {
```

Comment:
Test method name doesn't match the test input strings. Should it just be named `testNoErrorsReportedWhenCompanyNumberIsValid`.

Reply:
Changed name